src/components/Camera/index.js

```diff
@@ -16,7 +14,7 @@ import style from './style.scss'
 // support the precise resolution.
 const DEFAULT_CAMERA_HEIGHT_IN_PX = 720
 
-export type Props = {
+/* export type Props = {
```

Comment:
Why are the flow types here and in the other files commented out? Are you going to use them as reference for the typescript implementation?

Reply:
Sorry I forgot to explain in the description: the TypeScript transpiler forbids the uses of types in JS files. Hence we have to _hide_ them and hopefully can recycle when we convert them into TypeScript.